editor.planx.uk/src/ui/editor/ListManager/ListManager.tsx

```diff
@@ -38,9 +41,44 @@ export interface Props<T, EditorExtraProps = {}> {
 
 const Item = styled(Box)(({ theme }) => ({
   display: "flex",
-  marginBottom: theme.spacing(2),
+  "&:last-child": {
+    marginBottom: theme.spacing(2),
+  },
 }));
 
+const InsertButtonRoot = styled(ButtonBase)(({ theme }) => ({
+  justifyContent: "space-between",
+  paddingLeft: theme.spacing(1.5),
+  paddingRight: theme.spacing(3),
+  width: "100%",
+  height: theme.spacing(2),
+  opacity: 0,
+  transition: "opacity 0.4s cubic-bezier(0.4, 0, 0.2, 1)",
+  "&:hover": {
+    opacity: 1,
+  },
```

Comment:
Setting `visibility` doesn't work here as this removes the `":hover"` interaction.